templates/vhost/_directories.erb

```diff
@@ -235,7 +235,10 @@
       <%- end -%>
     <%- end -%>
     <%- if directory['dav'] -%>
-    Dav <%= directory['dav'] %>
+    DAV <%= directory['dav'] %>
```

Comment:
Did you switch from `Dav` to `DAV` on purpose?

Reply:
Wanted to enable the DAV svn in the location with SVNParentPath

Reply:
Seems like its working on 'Dav svn'  too. Have removed that changes